### PR TITLE
Typos

### PR DESCRIPTION
**Pull request by dp111**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -95,7 +95,7 @@ rpiboot (20210714~083443) buster; urgency=medium
   * Don't load bootcode from the overlay directory
   * pieeprom-2021-01-16: Update to latest release for BCM2711 XHCI boot
   * Update EEPROM image to latest CM4 manufacturing image + configuration
-  * msd: CM4: Fix occasionaly USB errors
+  * msd: CM4: Fix occasionally USB errors
```
Comment (pelwell) [inline debian/changelog:128]: Nope - `occasional`, surely.

**Comment by timg236** [comment]
Except for the comment by @pelwell it looks good to me.

File: debian/changelog (hunk 1 above)
Comment (dp111) [reply debian/changelog]: Fixed

**Comment by pelwell** [comment]
Thanks!

**Pull request merged**